server/pbench/bin/gold/test-7.8.txt

```diff
@@ -1039,7 +1039,19 @@ len(actions) = 19
         },
         "_type": "pbench-tool-data-iostat"
     }
-]
+]could not convert binary: /var/tmp/pbench-test-server/tmp/uperf__2016-10-06_16:34:03/1-tcp_stream-16384B-32i/sample1/tools-default/flat8_40gb/sar/sar.data
```

Comment:
so there's a separate test-11 that is evaluated to check sar-index work. uperf test contains sar data and that is being detected as a part of the new index-pbench code that was pushed in this PR. Either we should specify a flag (--disable-sar ?) to index pbench or we should remove this data in uperf. Removing it implies sar is not really a part of uperf results (but which it is!). But for unit tests, it sounds reasonable and IIUC that's what you meant here?